src/main/java/ru/itmo/sd/deadliner2bot/service/TodoService.java

```diff
@@ -0,0 +1,39 @@
+package ru.itmo.sd.deadliner2bot.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.itmo.sd.deadliner2bot.model.Todo;
+import ru.itmo.sd.deadliner2bot.repository.TodoRepository;
+
+import java.time.LocalDateTime;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class TodoService {
+
+    private final TodoRepository todoRepository;
+
+    public List<Todo> findAllTodosByChatId(long chatId, LocalDateTime now) {
```

Comment:
Same as above, repository and service names should reflect that only not yet completed TODOs are fetched.